Assets/Scripts/RoundController.cs

```diff
@@ -37,6 +40,9 @@ public class RoundController : MonoBehaviour {
     public void endRound() {
         Debug.Log("Ending round"); // TODO: transfer scores etc.
         Destroy(board);
+        gameObject.GetComponent<ObjectGenerator>().enabled = false;
```

Comment:
This doesn't stop the coroutines that run on seperate threads to spawn object (causing error messages when they try to access the board). I suggest solving it from the object generator instead, as done in #71.

Reply:
I still get errormessages when running #71, but not when i run this. This might be unity and not the code though.

Reply:
After leaving the code running i now see that I'm getting the error still. I'll remove this line then